pom.xml

```diff
@@ -111,6 +112,11 @@
                 <version>${mockserver.version}</version>
                 <scope>test</scope>
             </dependency>
+            <dependency>
+                <groupId>ch.qos.logback</groupId>
+                <artifactId>logback-classic</artifactId>
+                <version>${logback.version}</version>
```

Comment:
@sunyuhan1998 Can you put <scope>test</scope> here? I think this is only for tests, so it will help make sure